package.json

```diff
@@ -31,6 +30,7 @@
     "test-compiled": "npm run prepublish && NODE_PATH=./lib mocha --compilers js:babel-register --recursive tests/src",
     "test-all": "npm test && for resolver in ./resolvers/*; do cd $resolver && npm test && cd ../..; done",
     "prepublish": "npm run build",
+    "prepublishOnly": "npm run copy-metafiles",
```

Comment:
why is this changed? we want it running after the build step, not just on prepublish.

Reply:
Why do you want it to run after the build step? From what it does, it seems to be only necessary or at all meaningful if done before packaging.

The reason for limiting  the `copy-metafiles` script to `prepublishOnly` is that it throws a syntax error if run on Windows (because it's a `sh` script), one of the two reasons the tests don't run on Appveyor CI right now.

This is a quick fix that only prevents publishing from Windows versus preventing to publish or even to do a clean `npm install`.

Reply:
I fixed this by adding `in-publish` rather than using the hacky `prepublishOnly` or by breaking `npm run build`.